internal/command/volumes/extend.go

```diff
@@ -72,11 +74,29 @@ func runExtend(ctx context.Context) error {
 		return err
 	}
 
-	sizeGB := flag.GetInt(ctx, "size")
+	sizeFlag := flag.GetString(ctx, "size")
+	sizeGB, err := strconv.Atoi(sizeFlag)
+	if err != nil {
+		sizeBytes, err := units.FromHumanSize(sizeFlag)
+		if err != nil {
+			return fmt.Errorf("invalid size: %w", err)
+		}
+
+		sizeGB = int(sizeBytes) / units.GB
+	}
```

Comment:
what about moving this parsing logic to `helpers/units.go` and reuse it for `fly vol extend` and `fly scale memory` ? 

There is another commonly used flag name `--vm-size` usually parsed by `internal/flag/machines.go`'s `GetMachineGuest()` that could reuse the helper

Reply:
And pass in the unit of division

Reply:
`GetMachineGuest` can't reuse the helper as the size is set as `string`.